src/remoteConnector.ts

```diff
@@ -615,13 +632,13 @@ export default class RemoteConnector extends Disposable {
 
 		const copy = 'Copy';
 		const configureSSH = 'Configure SSH';
-		const action = await vscode.window.showInformationMessage(`An SSH key is required for passwordless authentication.\nAlternatively, copy and use this password: ${maskedPassword}`, { modal: true }, copy, configureSSH);
+		const action = await vscode.window.showInformationMessage(`You don't have registered any SSH public key for this machine in your Gitpod account.\nAlternatively, copy and use this password: ${maskedPassword}`, { modal: true }, copy, configureSSH);
```

Comment:
Maybe let's change to `showWarningMessage`.

Users should feel that it is not OK to use password and they should move to use SSH keys.

I think we should be also specific that password is generated for each workspace instance. Can we add something like `Password resets on every workspace restart.`?